### ISSUE TITLE
"Cannot find module" shows weird path prefix: \\?\

### ISSUE THREAD

**Issue by Janneman84** [electron/electron#10324]
I have an Electron app that uses a native node module. It works fine everywhere except on my colleague's Windows 10 computer. He gets the error "Cannot find module. \\\\?\C:\Users\...". The path that follows is correct, however it shows "\\\\?\" in front of it.

I renamed the folder to trigger the error on my own W10 pc and here it doesn't show the weird "\\\\?\" prefix in the path in the error message.

I tried moving the app folder to the root of C:\ but no luck. I believe the \\\\?\ is to blame, any idea where that comes from?

**Comment by enlight** [comment]
`\\?\` is an extended-length path prefix that you can read about on [MSDN](https://msdn.microsoft.com/en-us/library/windows/desktop/aa365247%28v=vs.85%29.aspx#maxpath), so it should be a perfectly valid path. I don't know why the module isn't found on your colleague's machine.

**Comment by Janneman84** [comment]
Pretty weird indeed. I tried running as Administrator too.

The .exe gave me a Smartscreen warning, so I tried disabling Windows Defender to be sure. Doesn't seem to make a difference either...

**Comment by Janneman84** [comment]
Turns out there was a dll file missing that caused the module to not work. I found the error message pretty misleading, saying that the file doesn't exist while there was in fact a different problem.

More specific info on the dlls can be found here: https://github.com/NodeRT/NodeRT/issues/87

Electron is already including 2 of the 4 VC redist dlls, maybe it's a good idea to include the other two as well to prevent issues like this (check the link)?
